integration-tests/plugins/__tests__/services/product.ts

```diff
@@ -0,0 +1,147 @@
+import path from "path"
```

Comment:
comment(this-file): Something is off about having integration tests for core services in the plugins directory. I think we should move this to API (even though not ideal either) to have it closer to the core. Wdyt?

Reply:
agree, however there's something about having the redis_url in the project config of the api that breaks the `bootstrapApp({ cwd })` method for server setup, though it's needed to get the container